.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/pkp/pkp-lib.git
 [submodule "plugins/generic/usageStats"]
         path = plugins/generic/usageStats
-        url = https://github.com/pkp/usageStats.git
+        url = https://github.com/asmecher/usageStats.git
```

Comment:
Should this change be in these commits?

Reply:
That's in a `DO NOT MERGE` commit (https://github.com/pkp/ojs/pull/2710/commits/604320113046fe0595af96ce4f13c07ec1c4e143). I'm just trying to get the tests to pass, without total success yet. I'll rebase that change out of existence before merging.